watir-ng.gemspec

```diff
@@ -18,10 +18,10 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency 'watir', '~> 6.0.0'
+  spec.add_runtime_dependency 'watir', '~> 6.0'
   
   spec.add_development_dependency "bundler"
   spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "rspec", "~> 3.2.0"
-  spec.add_development_dependency "cucumber", "~> 2.0.0"
+  spec.add_development_dependency "rspec", "~> 3.0"
```

Comment:
Moving from `3.2.0` -> `3.0` seems odd. Why the downgrade in minor version? If this were `3.2.0` -> `3.2`, I'd understand. As is, I'll need some details to accept this.

Reply:
Setting it to 3.0 rather than 3.2.0 should allow for the newer rspec versions such as 3.6. Setting the gemspec dependency to include all 3 digits was causing the gems to cap out at 3.2 for rspec, 2.0.2 for cucumber and 6.0.3 for watir. For rspec in particular, if you'd prefer that I resubmit with the filter set to "~> 3.2", I can do so.